setup.py

```diff
@@ -1,12 +1,12 @@
 from setuptools import setup, find_packages
 import sys
 from pathlib import Path
-from iblvideo.__init__ import __version__
+# from iblvideo.__init__ import __version__
```

Comment:
Should this be commented?

Reply:
yep good catch, I think Julia did this in the beginning for some debugging purpose and I never changed it back; fixed now!